content/docs/reference/design-docs/stdcm/pathfinding_module/standard_allowance.en.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Standard allowance"
+linkTitle: "5 - Standard allowance"
+weight: 50
+---
+
+
+The STDCM module must be usable with
+[standard allowances]({{< ref "docs/explanation/running_time_calculation/allowances" >}} "marge de régularité").
+The user can set an allowance value, expressed either as a function of
+the run time or the travelled distance. This time must be added to the
+run time, so that it arrives later compared to its fastest possible
+run time.
+
+> For example: the user can set a margin of 5 minutes per 100km.
+> On a 42km long path that would take 10 minutes at best,
+> the train should arrive 12 minutes and 6 seconds after leaving.
+
+This can cause problems to detect conflicts, as an allowance would move
+the end of the train slot to a later time.
+The allowance must be considered when we compute conflicts as
+the graph is explored.
+
+The allowance must also follow the [MARECO](/pdf/MARECO.pdf) model.
```

Comment:
A very short explaination about MARECO here would be helpful without having to read the entire document (especially cause it's only in french !)